bold-hour-bebas/usr/share/asteroid-launcher/watchfaces/bold-hour-bebas.qml

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2018 - Timo Könnecke <el-t-mo@arcor.de>
+ *               2016 - Sylvia van Os <iamsylvie@openmailbox.org>
+ *               2015 - Florent Revest <revestflo@gmail.com>
+ *               2012 - Vasiliy Sorokin <sorokin.vasiliy@gmail.com>
+ *                      Aleksey Mikhailichenko <a.v.mich@gmail.com>
+ *                      Arto Jalkanen <ajalkane@gmail.com>
+ * All rights reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// Asteroid theme colors
+// yellow	Qt.rgba(0.945, 0.769, 0.059, 1)
+// Orange	Qt.rgba(1, 0.549, 0.149, 1)
+// red		Qt.rgba(0.871, 0.165, 0.102, 1)
+
+import QtQuick 2.1
+
+Item {
```

Comment:
Just to let you know, I find that watchface "neutral" enough to be upstreamed in asteroid-launcher and included by default in every build.
However, the screenshot you took of this watchface looks like the font is not being rendered correctly by the Text elements. Have you had a chance to debug that? (ie: the QML's Text element has different rendering options)

Reply:
Will definately have a look. It did not bother me enough to debug until now :)

Reply:
Got it! It is nicely rendered when changing renderType to Native:
`renderType: Text.NativeRendering`